### ISSUE TITLE
[FEEDBACK] Update dev-node tutorial + github wiki

### ISSUE THREAD

**Issue by cpengilly** [ethereum-optimism/docs#118]
## Brief Description of the Docs Request
Feedback from devconnect about dev-node tutorial

## Description of the Documentation You'd Like
I'll point out some minor issues in the docs that we noticed during setup:

- The [optimism docs](https://community.optimism.io/docs/developers/build/dev-node/#) do not mention that you need to build the bedrock contracts prior to the deployment (only the [github wiki](https://github.com/ethereum-optimism/optimism/blob/65ec61dde94ffa93342728d324fecf474d228e1f/specs/meta/devnet.md) has this info).
- The contracts are located in the packages/bedrock-contracts, while the github wiki instructs to build from packages/contracts (does not exist).
- When building the contracts using yarn (as per the guidelines in the docs), we were prompted to use pnpm instead. Using pnpm the installation and build process worked fine.

## Links to Related Pages
n/a

## Additional Context
dev-node tutorial and github wiki need to be updated to match

**Comment by cpengilly** [comment]
@smartcontracts pinging you to see if we can move this up in the priority list.

**Comment by cpengilly** [comment]
adding more feedback from different issue (going to close duplicate issue): 
First, I wanted to work with the OP devnet. I followed the instructions at https://community.optimism.io/docs/developers/build/dev-node/, but wasn't successful. I discovered via Discord and GitHub issues that there are several missing steps, including git submodule init && git submodule update. The docs page still doesn't reflect that.
